docs/about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Ibhicheckesho | About</title>
+  </head>
+  <body>
+    <header id="header-about">
+      <div class="container">
+        <nav id="main-nav">
+          <div class="container">
```

Comment:
- [ ] Please, consider that the social media navigation bar is still not added at the top. Kindly ask to fix it, as requested by the previous reviewer. When you scroll the page, you can see an empty space for it. Please, look at the screen below:
![2022-02-03 11 59 43](https://user-images.githubusercontent.com/49064106/152304290-6010f10a-14b4-4f2e-bbd7-d5bfe05983a8.gif)

Reply:
![image](https://user-images.githubusercontent.com/22321030/152308255-9a849b15-d878-4f34-bdff-2f68dfebf2b2.png)
I do not know what heppened but the social links are there

Reply:
- [ ] This is not yet fixed. Kindly check the `about` page to see this bug as the one on the homepage displays properly.